app/assets/javascripts/poller.js

```diff
@@ -5,15 +5,37 @@
     init: function() {
       this.$poller = $('.js-poller');
       this.$messageForm = $('.js-message-form');
+      this.pollingTimeout = 1000 * 60 * 60; // 1 hour
 
       if (this.$poller.length && this.$messageForm.length) {
         this.setLast(this.timestamp());
+        this.bindEvents();
         this.start();
       }
     },
 
+    pageVisibilitySupported: function() {
+      return 'hidden' in document ? true : false;
```

Comment:
Don't think you need the ternary here? Would `return 'hidden' in document` work?